examples/todomvc/src/index.js

```diff
@@ -1,12 +1,48 @@
 import React from 'react'
 import { render } from 'react-dom'
-import { createStore } from 'redux'
+import { createStore, applyMiddleware, compose } from 'redux'
+import { genWcStoreEnhancer } from './wildcard/storeEnhancer'
 import { Provider } from 'react-redux'
 import App from './components/App'
 import reducer from './reducers'
 import 'todomvc-app-css/index.css'
+import { addTodo, editTodo, completeTodo } from './actions'
 
-const store = createStore(reducer)
+// We define a function which converts a Wildcard generic action
+// into a domain-specific actions for our app, if applicable.
+// Otherwise just returns the original action.
+// todo: what helpers can we provide here to make this easier?
+const actionTransformer = (action) => {
```

Comment:
Define logic for how generic table actions should convert into specific actions in this app -- eg, create/edit getting mapped to existing actions